pom.xml

```diff
@@ -130,62 +178,91 @@
             <artifactId>commons-collections</artifactId>
             <version>3.2.2</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>${guava.version}</version>
-        </dependency>
         <dependency>
             <groupId>org.dhatim</groupId>
             <artifactId>dropwizard-sentry</artifactId>
-            <version>2.1.2-4</version>
+            <version>2.1.6</version>
         </dependency>
         <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
             <version>20240205</version>
         </dependency>
-        <!-- testing -->
         <dependency>
-            <groupId>uk.gov.service.payments</groupId>
-            <artifactId>testing</artifactId>
-            <version>${pay-java-commons.version}</version>
-            <scope>test</scope>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+        </dependency>
```

Comment:
Do we need this and the FindBugs JSR 305 dependency?

Reply:
No